x11-terms/cool-retro-term/cool-retro-term-1.2.0.ebuild

```
@@ -0,0 +1,58 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils xdg
+
+QTW_PN="qmltermwidget"
+QTW_PV=63228027e1f97c24abb907550b22ee91836929c5
+QTW_P="${QTW_PN}-${QTW_PV}"
+
+DESCRIPTION="Terminal emulator with an old school look and feel"
+HOMEPAGE="https://github.com/Swordfish90/cool-retro-term/"
+
+SRC_URI="
+	https://github.com/Swordfish90/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/Swordfish90/${QTW_PN}/archive/${QTW_PV}.tar.gz -> ${QTW_P}.tar.gz
+"
+
+LICENSE="GPL-2 GPL-3"
```

```suggestion
LICENSE="GPL-2 GPL-3+"
```

> * cool-retro-term is free software: you can redistribute it and/or modify
> * it under the terms of the GNU General Public License as published by
> * the Free Software Foundation, either version 3 of the License, or
> * (at your option) any later version.

Could you verify GPL-2 is still found somewhere and that it's not GPL-2+?

---

This should be fix. In fact, `cool-retro-term` is directly under GPL-3+ and the built-in `qmltermwidget` is under BSD and GPL-2+.